### PR TITLE
Assorted tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
Please check the [non-whitespace diff](https://github.com/mozfreddyb/webext-firstpartyisolation/pull/18/files?w=1).

BTW, if there is a common Mozilla ESLint config we could use that.

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -1,68 +1,76 @@
...
+    updateIcon(details.levelOfControl !== 'not_controllable' ?
+      details.value :
+      false);
+  });
 }
+*/
 
+function setFPI(state) {
+  chrome.privacy.websites.firstPartyIsolate.set({ value: state }, () => {
+    if ((typeof chrome.runtime.lastError === 'undefined') || (chrome.runtime.lastError === null)) {
```
Comment (XhmikosR) [inline background.js]: We could probably change this to `!chrome.runtime.lastError`

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
 node_js:
   - 8
-before_script:
-  - npm install eslint
\ No newline at end of file
+cache:
+  directories:
+    - "$HOME/.npm"
+notifications:
+  email: false
```
Comment (mozfreddyb) [inline .travis.yml]: please leave this as true. I find them useful from time to time.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,18 @@
 {
   "name": "webext-firstpartyisolation",
   "version": "1.3.1",
+  "private": true,
```
Comment (mozfreddyb) [inline package.json]: Why is this necessary? :-) First time I see this

File: background.js (hunk 1 above)
Comment (mozfreddyb) [reply background.js]: Please do so :)

File: package.json (hunk 3 above)
Comment (XhmikosR) [reply package.json]: Since the package isn't published on npm, this flag prevents accidental publishing.

**Comment by XhmikosR** [comment]
@mozfreddyb: should be OK now.

**Comment by mozfreddyb** [comment]
Thank you @XhmikosR
